packages/openfeature-web-provider/src/ConfidenceWebProvider.ts

```diff
@@ -15,9 +15,11 @@ import equal from 'fast-deep-equal';
 import { ApplyManager, ConfidenceClient, Configuration, ResolveContext } from '@spotify-confidence/client-http';
 
 export interface ConfidenceWebProviderOptions {
-  apply?: {
-    timeout?: number;
-  };
+  apply:
+    | {
+        timeout: number;
```

Comment:
Should we actually have min-value for timeout? Can generate a lot of requests and it will be difficult to enforce later. I actually think 250ms is a good minimum. Maybe we should even have it non configurable for now. I.e. just `backend` or `client`

Reply:
After a sync, we decided to keep it simple for now and use client and backend. where client has a timeout of 250.